server/src/main/java/io/crate/expression/reference/sys/shard/ShardRowContext.java

```diff
@@ -58,7 +56,6 @@ public class ShardRowContext {
     private final String path;
     @Nullable
     private final String blobPath;
-    @Nullable
```

Comment:
Shouldn't this stay? It is set to `null` sometimes, https://github.com/crate/crate/blob/3d1d0c44f6453a8f6d80c2fda6af10aae91a78cc/server/src/main/java/io/crate/expression/reference/sys/shard/ShardRowContext.java#L100

Reply:
yep, reverted, thx!